.github/workflows/build-release.yml

```diff
@@ -123,6 +152,9 @@ jobs:
       - name: Zip Linux bundle
         run: tar -czvf SyncVault-${{ github.ref_name }}-linux.tar.gz exec/linux
 
+      - name: Zip macOS app
+        run: tar -czvf SyncVault-macos.tar.gz exec/macos
```

Comment:
Like in linux and windows, name the tar file as -
`SyncVault-${{ github.ref_name }}-macos.tar.gz`